rocky/tools/models.py

```diff
@@ -157,6 +171,7 @@ class STATUSES(models.TextChoices):
 
     user = models.ForeignKey("account.KATUser", on_delete=models.PROTECT, related_name="members")
     organization = models.ForeignKey(Organization, on_delete=models.CASCADE, related_name="members")
+    groups = models.ManyToManyField(Group, related_name="members", blank=True)
```

Comment:
I think "members" as related name for groups is confusing as it won't really give the members of the group, but the relation between organization, user and group. I think the default organizationmember_set would be fine if we would use it in the code.